TMTumblrSDK.podspec.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "TMTumblrSDK",
-  "version": "5.4.0",
+  "version": "5.4.1",
```

Comment:
I don't think if there's anyone else is using this library(or no?) so I only incremented the patch version. But in fact, if we want to follow semantic versioning rules we might want to update it as 6.0.0 instead. Because this PR changes a public method and a public protocol, and both are breaking changes. Let me know if you think this should be changed.